core/src/sumeragi/fault.rs

```diff
@@ -278,7 +287,7 @@ impl<F: FaultInjection> SumeragiWithFault<F> {
                 return;
             }
 
-            let reconnect_in_ms = 100;
+            let reconnect_in_ms = 250;
```

Comment:
Any particular reason for this change?

Reply:
The timings needed to be changed for the test to pass.